src/storage.ts

```diff
@@ -117,30 +131,30 @@ export interface AutoclassConfig {
   terminalStorageClass?: 'NEARLINE' | 'ARCHIVE';
 }
 
-export interface CreateBucketRequest {
+export interface CreateBucketRequest extends BucketMetadata {
   archive?: boolean;
-  autoclass?: AutoclassConfig;
   coldline?: boolean;
-  cors?: Cors[];
-  customPlacementConfig?: CustomPlacementConfig;
+  dataLocations?: string[];
   dra?: boolean;
   enableObjectRetention?: boolean;
-  hierarchicalNamespace?: {
-    enabled?: boolean;
-  };
-  iamConfiguration?: {
-    publicAccessPrevention?: string;
-    uniformBucketLevelAccess?: {
-      enabled?: boolean;
-      lockedTime?: string;
-    };
-  };
-  location?: string;
   multiRegional?: boolean;
   nearline?: boolean;
+  predefinedAcl?:
+    | 'authenticatedRead'
+    | 'private'
+    | 'projectPrivate'
+    | 'publicRead'
+    | 'publicReadWrite';
+  predefinedDefaultObjectAcl?:
+    | 'authenticatedRead'
+    | 'bucketOwnerFullControl'
+    | 'bucketOwnerRead'
+    | 'private'
+    | 'projectPrivate'
+    | 'publicRead';
+  projection?: 'full' | 'noAcl';
   regional?: boolean;
   requesterPays?: boolean;
-  retentionPolicy?: object;
```

Comment:
will removing this cause any issues?

Reply:
It isn't removed. This interface now extends `BucketMetadata` https://github.com/googleapis/nodejs-storage/blob/18eef67892af6c7311a12d4c22579e240149eb02/src/bucket.ts#L347